Common++/header/MacAddress.h

```diff
@@ -40,21 +40,21 @@ namespace pcpp
 		 * @todo there is no verification array length >= 6. If this is not the case, address will read uninitialized memory
 		 * @param[in] addr A pointer to the byte array containing 6 bytes representing the MAC address
 		 */
-		MacAddress(const uint8_t* addr) : m_IsValid(true) { memcpy(m_Address, addr, sizeof(m_Address)); }
+		explicit MacAddress(const uint8_t* addr) : m_IsValid(true) { memcpy(m_Address, addr, sizeof(m_Address)); }
 
 		/**
 		 *  A constructor that creates an instance of the class out of a (char*) string.
 		 *  If the string doesn't represent a valid MAC address, instance will be invalid, meaning isValid() will return false
 		 *  @param[in] addr A pointer to the (char*) string
 		 */
-		MacAddress(const char* addr) { init(addr); }
+		explicit MacAddress(const char* addr) { init(addr); }
 
 		/**
 		 *  A constructor that creates an instance of the class out of a std::string.
 		 *  If the string doesn't represent a valid MAC address, instance will be invalid, meaning isValid() will return false
 	 	 *	@param[in] addr A pointer to the string
 		 */
-		MacAddress(const std::string& addr) { init(addr.c_str()); }
+		explicit MacAddress(const std::string& addr) { init(addr.c_str()); }
```

Comment:
Maybe this c'tor should not be `explicit`?

That way we can do something like this:

```cpp
void foo(const MacAddress& mac)
{
    std::cout << mac;
}

int main()
{
    foo("00:00:5e:00:53:af");
    return 0;
}
```

Reply:
The same goes for constructors in `IpAddress.h` which I have excluded for now in #1003 to double check. Probably excluding both `MacAddress.h` and `IpAddress.h` might do the trick to make the functions easier to use.